cloudconfig/userdatacfg_win.go

```diff
@@ -141,6 +144,10 @@ func (w *windowsConfigure) ConfigureJuju() error {
 	return w.addMachineAgentToBoot()
 }
 
+func (w *windowsConfigure) ConfigureCustomOverrides() error {
+	return nil
```

Comment:
Can we add a todo here as i think we need to ensure windows is supported

Reply:
added